solutions/standard/variables.tf

```diff
@@ -0,0 +1,114 @@
+########################################################################################################################
+# Common variables
+########################################################################################################################
+
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The API Key to use for IBM Cloud."
+  sensitive   = true
+}
+
+variable "existing_resource_group" {
+  type        = bool
+  description = "Whether to use an existing resource group."
+  default     = false
+}
+
+variable "resource_group_name" {
+  type        = string
+  description = "The name of a new or an existing resource group in which to provision KMS resources to."
+}
+
+variable "region" {
+  type        = string
+  description = "The region in which to provision KMS resources. If using existing KMS, set this to the region in which it is provisioned in."
+  default     = "us-south"
+}
+
+########################################################################################################################
+# Secrets Manager
+########################################################################################################################
+
+variable "secrets_manager_instance_name" {
+  type        = string
+  description = "The name to give the Secrets Manager instance that will be provisioned by this solution."
+  default     = "base-security-services-sm"
+}
+
+variable "service_plan" {
+  type        = string
+  description = "The service/pricing plan to use when provisioning a new Secrets Manager instance. Allowed values: 'standard' and 'trial'. Only used if `provision_sm_instance` is set to true."
+  default     = "standard"
+  validation {
+    condition     = contains(["standard", "trial"], var.service_plan)
+    error_message = "Allowed values for sm_service_plan are \"standard\" and \"trial\"."
+  }
+}
+
+variable "service_endpoints" {
+  # public-and-private until IBM Console connects to SM via private endpoints
+  type        = string
+  description = "The service endpoints to enable for all services deployed by this solution. Allowed values are `private` or `public-and-private`. If selecting `public-and-private`, communication to the instances will all be done over the public endpoints. Ensure to enable virtual routing and forwarding (VRF) in your account if using `private`, and that the terraform runtime has access to the the IBM Cloud private network."
```

Comment:
You might want to tweek - this DA only provisions SM service instance

Reply:
would i need to change the main module? https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager/blob/5dd20e7cf737b7106e0bbce7c7628371242f67c6/main.tf#L27